src/NServiceBus.Core/IMessageSession.cs

```diff
@@ -35,8 +35,8 @@ public interface IMessageSession
         /// </summary>
         /// <typeparam name="T">The type of message, usually an interface.</typeparam>
         /// <param name="messageConstructor">An action which initializes properties of the message.</param>
-        /// <param name="publishOptions">Specific options for this event.</param>
-        Task Publish<T>(Action<T> messageConstructor, PublishOptions publishOptions);
+        /// <param name="options">Specific options for this event.</param>
+        Task Publish<T>(Action<T> messageConstructor, PublishOptions options);
```

Comment:
Given https://github.com/Particular/NServiceBus/pull/5772 should we roll this back?